src/main/java/frc/robot/subsystems/climber/ClimberIO.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.climber;
+
+import org.littletonrobotics.junction.AutoLog;
+
+/** Generic subsystem hardware interface. */
+public interface ClimberIO {
+  /** Contains all of the input data received from hardware. */
+  @AutoLog
+  public static class ClimberIOInputs {
+    double leftVelocityRPM = 0.0;
+    double leftPosition = 0.0;
+    double leftStatorCurrentAmps = 0.0;
+    double leftSetpoint = 0.0;
+
+    double rightVelocityRPM = 0.0;
+    double rightPosition = 0.0;
+    double rightStatorCurrentAmps = 0.0;
+    double rightSetpoint = 0.0;
+
+    double leftReferenceVelocity = 0.0;
```

Comment:
I don't think we will do velocity control for the climber; so, these can be removed.